_includes/example-issued-alumni-credential.html

```diff
@@ -0,0 +1,45 @@
+<div class="ui basic segment">
+  <details class="ui accordion">
+    <summary class="title">
+      <i class="dropdown icon"></i>
+      Example Generated / Issued Verifiable Credential
+    </summary>
+    <div class="content ui form">
+      <textarea class="field">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1",
+    "https://contexts.vcplayground.org/examples/alumni/v1.json",
+    "https://w3id.org/security/suites/ed25519-2020/v1"
```

Comment:
This is a good opportunity to start using the new data integrity suites.

Reply:
FWIW, these are direct copy/paste from the playground. I'd prefer to just update these when the playground examples updates...if that's OK.